source/ui/ProjectCreationScreen.qml

```diff
@@ -37,156 +45,212 @@ Item {
+        id: syntaxSelector
         anchors {
-            top: textInputName.bottom
-            left: parent.left
-            right: parent.horizontalCenter
-
-            topMargin: marginHeight
-            bottomMargin: marginHeight
-            leftMargin: marginWidth
-            rightMargin: marginWidth
+          right: parent.right
+          rightMargin: Theme.createProject.section.margin
+          verticalCenter: parent.verticalCenter
         }
-        model: ui.getArchitectures();
-        onCurrentIndexChanged: {
-          var currentArchitecture = model[currentIndex];
-          optionSelector.model = ui.getOptionNames(currentArchitecture);
-          parserSelector.model = ui.getParsers(currentArchitecture);
+        style: ComboBoxStyle {
+          font.pixelSize: Theme.createProject.section.fontSize
         }
+      }
     }
 
-    Text {
-      id: parserText
-      anchors {
-        left: parserSelector.left
-        bottom: parserSelector.top
+    ////////////
+    // Version
+    ////////////
+
+    ProjectCreationSection {
+      id: version
+      anchors.top: syntax.bottom
+
+      Text {
+        id: versionLabel
+        font.pixelSize: Theme.createProject.section.fontSize
+        text: "Version:"
+        anchors {
+          left: parent.left
+          verticalCenter: parent.verticalCenter
+          leftMargin: Theme.createProject.section.margin
+        }
       }
-      text: "Parser"
-    }
 
-    //choose the parser
-    ComboBox {
-      id: parserSelector
-      anchors {
-        top: memorySizeSelector.bottom
-        left: parent.horizontalCenter
-        right: parent.right
-
-        topMargin: marginHeight
-        bottomMargin: 0
-        leftMargin: marginWidth
-        rightMargin: marginWidth
+      ComboBox {
+        id: versionSelector
+        anchors {
+          right: parent.right
+          rightMargin: Theme.createProject.section.margin
+          left: versionLabel.right
+          leftMargin: Theme.createProject.section.margin
+          verticalCenter: parent.verticalCenter
+        }
+        style: ComboBoxStyle {
+          font.pixelSize: Theme.createProject.section.fontSize
+        }
       }
     }
 
-    Text {
-      id: optionSelectorText
+    /////////////////////////////
+    // Project Creation Button
+    /////////////////////////////
+
+    Button {
+      id: button
       anchors {
-        left: optionSelector.left
-        bottom: optionSelector.top
+        top: version.bottom
+        horizontalCenter: parent.horizontalCenter
+        topMargin: Theme.createProject.button.marginTop
       }
-      text: "Select a version."
-    }
 
-    //select a formula for the architecture
-    ComboBox {
-        id: optionSelector
-        anchors {
-          top: architectureSelector.bottom
-          left: parent.left
-          right: parent.horizontalCenter
+      MouseArea {
+        anchors.fill: button
+        cursorShape: Qt.PointingHandCursor
```

Comment:
Do you use the MouseArea inside the button only for the cursorShape? I think it would be better to create a custom Button in that case to avoid overlaying an additional mouse area. It would also be great for reusability, as there is quite much styling in this button.